README.md

```diff
@@ -34,6 +34,7 @@ Add jfrog-client-js as a dependency to your package.json file:
       - [Getting Xray Version](#getting-xray-version)
       - [Scanning Bulk of Dependencies](#scanning-bulk-of-dependencies)
       - [Scanning a Dependency Tree with Consideration to the JFrog Project](#scanning-a-dependency-tree-with-consideration-to-the-jfrog-project)
+      - [Scanning a Dependency Tree with Consideration to the JFrog Watches](#scanning-a-dependency-tree-with-consideration-to-the-jfrog-watches)
```

Comment:
...with Consideration to the JFrog Watches --> ...with Consideration to Xray Watches
(let's changes this in the actual section title as well).